lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -1169,6 +1169,7 @@ fn create_config(params: &ConnectParams, pin: String) -> Config {
         autologon: true,
         pointer_software_rendering: false,
         performance_flags: PerformanceFlags::default()
+            | PerformanceFlags::DISABLE_CURSOR_SHADOW
```

Comment:
I'd consider putting a comment on here so that we don't mistakenly remove the flag in the future.